digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/Individual.java

```diff
@@ -0,0 +1,42 @@
+package org.digit.health.sync.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Individual {
+    @JsonProperty("individualId")
+    private String individualId;
+
+    @JsonProperty("name")
+    private String name;
```

Comment:
Mrunal mentioned they might want to capture individual fields for fields like name (familyName, givenName, additionalName) and address etc.

Reply:
Made the suggested changes.